src/main/java/com/hubspot/jinjava/el/ext/JinjavaBeanELResolver.java

```diff
@@ -37,7 +38,7 @@ public class JinjavaBeanELResolver extends BeanELResolver {
     .add("wait")
     .build();
 
-  private static final Set<String> DEFERRED_EXECUTION_RESTRICTED_METHODS = ImmutableSet
+  private static final Set<String> DEFAULT_DEFERRED_EXECUTION_RESTRICTED_METHODS = ImmutableSet
```

Comment:
I think we can leave this as a static set for now. There's no use-case for making it configurable

Reply:
I remove this from config.